backend/backup_media.sh

```diff
@@ -1,3 +1,13 @@
 #!/bin/bash
 MYSQL_CONTAINER=$(docker ps --format "{{.Names}}" | grep mysql | grep orbitar)
```

Comment:
Can we make it
`docker-compose -f docker-compose.ssl.local.yml ps --format json | jq -r -M -c '.[] | select(.Service == "mysql") | .ID'`?